### PR TITLE
#23-Connect Everything Together - Christina, Co-authored-by: Irianni Munoz

### PR DESCRIPTION
**Pull request by christina-mifsud**
## Description

Fetching data from json to front-end

## Related to

Mention #23 

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -1,77 +1,49 @@
...
-				}
-				return res.json();
-			})
-			.then((body) => {
-				setMessage(body.message);
-			})
-			.catch((err) => {
-				console.error(err);
-			});
-	}, []);
```
Comment (mferryRV) [inline client/src/pages/Home.js:33]: I'd rather the data come from this useEffect than from a JSON file. If you pull that same data from the API, I'll approve

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -1,77 +1,49 @@
...
-    const handleShow = (videoInfo) => {
+
+	const handleClose = () => setShow(false);
+	const handleShow = (videoInfo) => {
 		setShow(true);
 		setSelectedIcon(videoInfo); // Use a state variable to store the selected video info
 	};
 
-	useEffect(() => {
-		fetch("/api")
```
Comment (mferryRV) [inline client/src/pages/Home.js:20]: What happens if you swap this to `/api/rhythms`?

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -7,71 +7,60 @@ import map from "./img/map.jpg";
...
-	const [message, setMessage] = useState("Loading...");
+
 	const [show, setShow] = useState(false);
 	const [selectedIcon, setSelectedIcon] = useState(null);
-    const handleClose = () => setShow(false);
-    const handleShow = (videoInfo) => {
+
+    const [rhythms, setRhythms] = useState([]);
+
+		const API_URL = "/api/rhythms";
```
Comment (mferryRV) [inline client/src/pages/Home.js:16]: What's going on with this indentation? I'd recommend cleaning this up.

File: client/src/pages/Home.js [hunk 4]
```diff
@@ -7,71 +7,60 @@ import map from "./img/map.jpg";
...
-	];
 
 	return (
 		<main role="main">
 			<div>
 				<img className="map" src={map} alt="North America Map" />
 
 				<div className="icons">
-					{dataList.map((dataItem) => (
+					{rhythms.map((dataItem) => (
```
Comment (mferryRV) [inline client/src/pages/Home.js:47]: Nice, love to see it

File: client/src/pages/rhythmsData.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+		"url": "https://www.youtube.com/watch?v=M6uM0qrjetQ",
+		"location": "Washington DC"
+	},
+	{
+		"id": 2,
+		"title": "Rumba Guaguanco",
+		"url": "https://www.youtube.com/watch?v=gJVT_5swkhA",
+		"location": "Havana, Cuba"
+	}
+]
```
Comment (mferryRV) [inline client/src/pages/rhythmsData.json:14]: This can be deleted

File: server/api.js [hunk 6]
```diff
@@ -1,18 +1,74 @@
...
-router.get("/rhythms", (req, res) => {
-	res.json(rhythms);
+
+// // Create a GET endpoint (rhythms)
+// router.get("/rhythms", (req, res) => {
+// 	res.json(rhythms);
+// });
+
+
+// HELP!! Cannot connect to DB!!
```
Comment (mferryRV) [inline server/api.js]: Remove these changes from this PR and let's just merge your work connecting with the API

**Review by mferryRV — COMMENTED** [review]
If you get rid of all the `server/` stuff we can merge this and do the server stuff in another PR

**Review by munozirianni1988 — APPROVED** [review]
Great, I'll aprove and work on the rest

**Pull request merged**